spec/lib/api/2.0/nodes-spec.js

```diff
@@ -857,4 +857,49 @@ describe('2.0 Http.Api.Nodes', function () {
         });
     });
 
+    describe('OBM support', function() {
+
+        var obm = {
+            service: 'noop-obm-service',
+            config: {
+                host: '1.2.3.4',
+                user: 'myuser',
+                password: 'mypass'
+            }
+        };
+
+
+        before(function() {
+            sinon.stub(nodesApi, 'getObmsByNodeId').resolves([]);
+            sinon.stub(nodesApi, 'putObmsByNodeId').resolves([]);
+        });
+
+        after(function() {
+            nodesApi.getObmsByNodeId.restore();
+            nodesApi.putObmsByNodeId.restore();
+        });
+
+        it('should call getObmsByNodeId', function() {
+            return helper.request().get('/api/2.0/nodes/123/obm')
+                .expect('Content-Type', /^application\/json/)
+                .expect(200)
+                .expect(function() {
```

Comment:
For GET API, the response body is a critical data, so need some assertion to judge the response data is correct.